packages/react-native/ReactCommon/react/renderer/mounting/MountingCoordinator.cpp

```diff
@@ -104,31 +104,35 @@ std::optional<MountingTransaction> MountingCoordinator::pullTransaction()
   }
 
   // Override case
-  auto mountingOverrideDelegate = mountingOverrideDelegate_.lock();
-  auto shouldOverridePullTransaction = mountingOverrideDelegate &&
-      mountingOverrideDelegate->shouldOverridePullTransaction();
-
-  if (shouldOverridePullTransaction) {
-    SystraceSection section2("MountingCoordinator::overridePullTransaction");
-
-    auto mutations = ShadowViewMutation::List{};
-    auto telemetry = TransactionTelemetry{};
-
-    if (transaction.has_value()) {
-      mutations = transaction->getMutations();
-      telemetry = transaction->getTelemetry();
-    } else {
-      number_++;
-      telemetry.willLayout();
-      telemetry.didLayout();
-      telemetry.willCommit();
-      telemetry.didCommit();
-      telemetry.willDiff();
-      telemetry.didDiff();
+  for(auto it = mountingOverrideDelegates_.begin(); it < mountingOverrideDelegates_.end(); it++)
+  {
+    auto &delegate = *it;
```

Comment:
```suggestion
  for (const auto &delegate : mountingOverrideDelegates_)
  {
```

Not sure about const